.rosinstall

```diff
@@ -0,0 +1,8 @@
+- git:
+    local-name: ros_model_parser
+    uri: https://github.com/ipa320/ros_model_parser.git
+    version: master
+- git:
+    local-name: rosgraph_monitor
+    uri: https://github.com/ipa-hsd/rosgraph_monitor.git
```

Comment:
why this?

Reply:
It is this one: https://github.com/ipa320/rosgraph_monitor/pull/1. Will update this once the PR is merged.